eng/pipelines/mgmt.yml

```diff
@@ -9,8 +9,7 @@ variables:
   skipComponentGovernanceDetection: true
   msBuildLogDir: msbuildlogs
   loggingArgs: '/clp:ShowtimeStamp /flp:LogFile=$(msBuildLogDir)/msbuild.normal.log;Verbosity=normal /flp1:Summary;Verbosity=minimal;LogFile=$(msBuildLogDir)/msbuild.sum.log /flp2:warningsonly;logfile=$(msBuildLogDir)/msbuild.wrn.log /flp3:errorsonly;logfile=$(msBuildLogDir)/msbuild.err.log'
-  RPScopeArgs: ''
-  #/p:PullRequestNumber=$(system.pullrequest.pullrequestnumber) /p:RepoHtmlUrl=https://github.com/$(build.repository.id) /p:CIBuildId=$(OfficialBuildId)'
+  RPScopeArgs: '/p:PullRequestNumber=$(system.pullrequest.pullrequestnumber) /p:RepoHtmlUrl=https://github.com/$(build.repository.id) /p:CIBuildId=$(OfficialBuildId)'
```

Comment:
The OfficialBuildId is never set anywhere for your yml so I don't expect that will ever be set correctly.

Reply:
@weshaggard I did set it at the end of the line for RPScopeArgs
/p:CIBuildId=$(OfficialBuildId)

Reply:
I see you setting CIBuildId but where is OfficialBuildId set?